lab/index.html

```diff
@@ -3,55 +3,79 @@
   <header>
```

Comment:
Your tags are off. This should be a <head> tag and the title and links should go in there. The <header> tag should be inside the body tag and should include what you currently have in the <head> tag.